nix/overlays/haskell-packages.nix

```diff
@@ -50,6 +50,15 @@ let
           }
           { };
 
+      warp =
+        lib.dontCheck (prev.callHackageDirect
```

Comment:
Just curious - what does the `lib.dontCheck` do?

I see we are using it on the `hasql-` packages - but we are not using it on the `wai-` packages.